components/table/index.jsx

```diff
@@ -69,7 +77,7 @@ class InfiniteTable extends React.Component {
             value={searchTerm}
           />
         )}
-        <Table>
+        <Table className={className}>
```

Comment:
You do not want to pass it using rest props 🙂

Reply:
true, thanks